build.py

```diff
@@ -514,7 +514,7 @@ def run_ci_historic_benchmark():
         "ipykernel",
         "nbconvert",
         "pandas",
-        "qiskit>=1.2.2,<1.3.0",
+        "qiskit>=1.2.2,<2.0.0",
```

Comment:
The requirements.txt file says `qiskit>=1.3.0` but this says `qiskit>=1.2.2`. Is this correct?

Reply:
Good catch! I had originally left this because I thought this determines what we try and download when the Python module is installed, but this is just the build script. This should be updated to 1.3.

Reply:
```suggestion
        "qiskit>=1.3.0,<2.0.0",
```